starlite/handlers/base.py

```diff
@@ -210,3 +210,24 @@ def _validate_handler_function(self) -> None:
         return annotations."""
         if not self.fn:
             raise ImproperlyConfiguredException("Cannot call _validate_handler_function without first setting self.fn")
+
+    @property
+    def identifier(self) -> str:
+        """
+
+        Returns:
+            A path like identifier for the route handler.
+        """
+        target = cast("Any", self.fn)
+        if not hasattr(target, "__qualname__"):
+            target = type(target)
+        return f"{type(self).__name__}::{target.__module__}.{target.__qualname__}"
```

Comment:
I am a bit unsure about using `type(self).__name__` here.

The user code will be depending on "internal" class names of Starlite (correct me if I am wrong but I haven't imported and  used HTTPRouteHandler in my code yet) and also Starlite internal code will have troubles accessing handlers which use custom user classes.

For example, right now I am experimenting with serving static files and related template functions. In short, I added a class derived from `starlette.static_files.StaticFiles` which introduces a bit of custom logic so I can host some of my static files on S3 and also I added a template function `asset('css', 'myfile.css')` which generates url to myfile.css taking into account "path" parameter in `StaticFilesConfig`.

Internally `asset` uses handler index to find all handlers for static files and inspects them.

Obviously with this change I won't be able to share this function with my other projects, not propose to include into Starlite.

Reply:
This should be the name of the current class. If you think it's a problem I will remove this.

Reply:
Another reason why I don't like it is that simple `{target.__module__}.{target.__qualname__}` works better because I can use [importlib](https://docs.python.org/3/library/importlib.html) and import class/function for inspection/instantiation without having to remove the prefix first.

Reply:
ok, done. I simplified it as well. Please review 😉